crates/turbopack-css/src/process.rs

```diff
@@ -878,11 +906,38 @@ impl swc_core::css::visit::Visit for CssValidator {
 }
 
 /// We only vist top-level selectors.
-impl lightningcss::visitor::Visitor<'_> for CssValidator {
+impl lightningcss::visitor::Visitor<'_> for CssModuleValidator {
     type Error = ();
 
     fn visit_types(&self) -> lightningcss::visitor::VisitTypes {
-        visit_types!(SELECTORS)
+        lightningcss::visitor::VisitTypes::all()
+    }
+
+    fn visit_property(
+        &mut self,
+        property: &mut lightningcss::properties::Property,
+    ) -> Result<(), Self::Error> {
+        match property {
+            lightningcss::properties::Property::GridArea(p) => {
+                for line in [&p.row_start, &p.row_end, &p.column_start, &p.column_end] {
+                    if let lightningcss::properties::grid::GridLine::Area { name } = line {
+                        self.used_grid_areas.insert(name.to_string());
+                    }
+                }
+            }
+
+            lightningcss::properties::Property::GridTemplateAreas(
```

Comment:
Is this handling that property too?

[grid-template](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template)